dask_planner/src/sql/logical/explain.rs

```diff
@@ -0,0 +1,31 @@
+use datafusion::logical_expr::logical_plan::Explain;
+pub use datafusion::logical_expr::LogicalPlan;
```

Comment:
```suggestion
use datafusion::logical_expr::{
    LogicalPlan,
    logical_plan::Explain
};
```
I had originally used the `pub` when the project was organized differently. This is no longer needed however. We can also more neatly organize the `use` statements.